ignite/pkg/cliui/view/accountview/account_test.go

```diff
@@ -0,0 +1,36 @@
+package accountview_test
+
+import (
+	"testing"
+
+	"github.com/ignite/cli/ignite/pkg/cliui/view/accountview"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccount_String(t *testing.T) {
```

Comment:
It might be good to use the other test naming convention for consistency:

```suggestion
func TestAccountString(t *testing.T) {
```

Reply:
good catch. fixed